### ISSUE TITLE
[Low] need user friendly ux -> Misc supported Home page dashboard widgets order

### ISSUE THREAD

**Issue by PeopleInside** [LiveHelperChat/livehelperchat#1016]
![1](https://cloud.githubusercontent.com/assets/5006150/25385863/9e3ec99a-29c5-11e7-86ca-214a700e825c.png)

![2](https://cloud.githubusercontent.com/assets/5006150/25385865/a2218e30-29c5-11e7-93c3-d4363e250df3.png)

As you can see in the screen if user have to restore **Home page dashboard widgets order** will be unable to do with the suggested text.

**Comment by remdex** [comment]
Yes i know, in the future will be done more user friendly UX, i'm just out of time :)
